### PR TITLE
Update eks clean script to include cwagent-operator-eks-integ-*

### PR DESCRIPTION
**Pull request by sky333999**
# Description of the issue
We have stale clusters from our integ tests that arent getting cleaned up.

# Description of changes
Update eks cleaner script to include missing regex.

# License
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

# Tests
N/A

# Requirements
_Before commit the code, please do the following steps._
1. Run `make fmt` and `make fmt-sh`
2. Run `make lint`

### CODE REVIEW

**Review by okankoAMZ — APPROVED** [review]

**Review by movence — APPROVED** [review]

**Review by jefchien — APPROVED** [review]

**Review by jefchien — DISMISSED** [review]

**Review by movence — APPROVED** [review]

File: tool/clean/clean_eks/clean_eks.go [hunk 1]
```diff
@@ -66,33 +67,39 @@ func terminateClusters(ctx context.Context, client *eks.Client) {
...
+			deleteNodegroupInput := eks.DeleteNodegroupInput{
+				ClusterName:   aws.String(cluster),
+				NodegroupName: aws.String(nodegroup),
 			}
-			deleteClusterInput := eks.DeleteClusterInput{Name: aws.String(cluster)}
-			_, err = client.DeleteCluster(ctx, &deleteClusterInput)
+			_, err := client.DeleteNodegroup(ctx, &deleteNodegroupInput)
 			if err != nil {
-				log.Printf("could not delete cluster %s err %v", cluster, err)
+				log.Printf("could delete node groups %s cluster %s err %v", nodegroup, cluster, err)
```
Comment (movence) [inline tool/clean/clean_eks/clean_eks.go:96]: missing "not"?

Comment (sky333999) [reply tool/clean/clean_eks/clean_eks.go]: Fixed

**Review by movence — APPROVED** [review]

File: tool/clean/clean_util.go [hunk 2]
```diff
@@ -7,6 +7,7 @@ import "time"
 
 const (
 	KeepDurationOneWeek        = KeepDurationOneDay * 7
+	KeepDurationFourDays       = KeepDurationOneDay * 4
```
Comment (zhihonl) [inline tool/clean/clean_util.go:10]: Just curious but is there a reason we pick 4 days over 7 days or we just pick this number for more frequent clenaups?

Comment (sky333999) [reply tool/clean/clean_util.go:10]: Yeah the latter. Just to make it more frequent.

**Pull request merged**